event_waiting_list/views/crm_lead.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_waiting_list_lead_tree" model="ir.ui.view">
+        <field name="model">crm.lead.event.waiting.list.tree</field>
+        <field name="model">crm.lead</field>
+        <field name="arch" type="xml">
+            <tree string="Waiting list">
+                <field name="name" string="Name"/>
+                <field name="email_from" />
+                <field name="phone" />
+                <field name="event_id" invisible="1" />
+            </tree>
+        </field>
+    </record>
+    <record id="view_waiting_list_lead_search" model="ir.ui.view">
+        <field name="model">crm.lead.event.waiting.list.search</field>
+        <field name="model">crm.lead</field>
+        <field name="arch" type="xml">
+            <search string="Search waiting list">
+                <field name="event_id"/>
+            </search>
+        </field>
+    </record>
+    <record model="ir.actions.act_window" id="action_waiting_list_from_event">
+        <field name="res_model">crm.lead</field>
+        <field name="name">Event Waiting List</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">tree</field>
+        <field name="context">{'search_default_event_id': active_id, 'default_event_id': active_id}</field>
+        <field name="domain">[('type', '=', 'event_waiting_list')]</field>
+        <field name="view_id" ref="event_waiting_list.view_waiting_list_lead_tree" />
+        <field name="search_view_id" ref="event_waiting_list.view_waiting_list_lead_search" />
+    </record>
```

Comment:
Add filter + group by on event to std crm.lead search

Reply:
if you want to keep this separated then add a new menu item and make it filterable by event.
ATM the only way to see waiting list for one event is to go on the event and click on the button.
What if I want to see WL related to 1 or more events at the same time?